tests/test_card.py

```diff
@@ -0,0 +1,8 @@
+from unittest import TestCase
+from card import Card
+
+
+class CardTestCase(TestCase):
+    def test_str_method_returns_text_for_card(self):
+        card = Card(v=1, s='Spades')
+        self.assertEqual(repr(card), '1 of Spades')
```

Comment:
This is an incredibly simple test. It mostly makes sure that the `__repr__` method on `Card` is doing what we think it's doing. When we get to coverage, that method will now be considered "covered". We'll get into that later.